ch6/README.md

```diff
@@ -0,0 +1,31 @@
+# DLDG: Chapter 6: Maintaining your Delta Lake
+> Note: The chapter exercises and notebooks are currently a work in progress.
+
+## Datasets
+* [Covid-19 NYC Dataset](https://github.com/delta-io/delta-docs/tree/main/static/quickstart_docker/rs/data/COVID-19_NYT)
```

Comment:
Thank you for adding this, I do wonder if we might later be able to create a setup script or notebook that runs these setup steps.

Reply:
The dataset reference is just to let people know where it came from. In this case, that dataset is part of the docker-quickstart image. @P7h made the suggestion since it requires less effort to get up and running for the readers. 

I like the idea of making it easier to attach additional datasets too. I was loosely thinking we could map the `datasets` directory from the root of the project for that purpose, some kind of scratch space. 

We can decide later though. But +1 on the setup script.